signalflow/computation.go

```diff
@@ -40,6 +40,24 @@ type Computation struct {
 	MetadataTimeout time.Duration
 }
 
+// ComputationError exposes the underlying metadata of a computation error
+type ComputationError struct {
+	Code      interface{}
```

Comment:
I have confirmed by looking at the backend code that `Code` will always be an `int`, `Message` will always be a `string`, and `ErrorType` will always be a `string`.  I think it is pretty safe using those types even though it isn't officially documented.  Would also make it easier to inspect without having to do a bunch of type assertions/switches in Go.